nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/repository/TestWriteAheadFlowFileRepository.java

```diff
@@ -64,6 +65,7 @@ public static void setupProperties() {
     }
 
     @Before
+    @After
```

Comment:
Depending on the order of the tests, if building the entire framework-core module, we could see failures because data has been written to the directory specified in the src/test/resources/nifi.properties; so this ensures that we just clean up after ourselves.